### PR TITLE
add roles to gke SA

### PR DESCRIPTION
**Pull request by anakovt**

### CODE REVIEW

File: modules/external-project-iam/vars.tf [hunk 1]
```diff
@@ -21,4 +21,19 @@ variable gcr_project_id {
...
-}
\ No newline at end of file
+}
+
+variable gke_service_account {
+  type        = string
+  description = "GKE service account email to add IAM roles in tribe project for"
+}
+
+variable gke_tribe_iam_roles {
```
Comment (sclausson) [inline modules/external-project-iam/vars.tf]: These variable names indicate that this module is becoming very specific to a certain use case -- and less generic. Is this a potential warning sign that we we have a hard time using it for other purposes in the future? Perhaps we shouldn't embed langauage like "tribe" and "gcr" and "gke" -- but instead  use more generic names?

**Review by sclausson — COMMENTED** [review]
Seem inline comments

File: modules/external-project-iam/vars.tf (hunk 1 above)
Comment (anakovt) [reply modules/external-project-iam/vars.tf]: yeah, I can change these names to something more generic, but what? :smile:

Comment (anakovt) [reply modules/external-project-iam/vars.tf]: @sclausson these variables are used in resources where you add specific roles to the specific projects for the gke sa. so I don't know how you can name it to use "more generic names" - I can't name them all as `iam_roles`. can you please tell me what exact names you want me to use?

Comment (anakovt) [reply modules/external-project-iam/vars.tf]: I'll remove "tribe" from var names and resources names

Comment (sclausson) [reply modules/external-project-iam/vars.tf]: Well what is the intention of "gke_tribe_iam_roles" within a module called "external project iam"?  Perhaps, generically, this should be called "external_project_iam_roles".

Comment (anakovt) [reply modules/external-project-iam/vars.tf]: it's a list of IAM roles that will be added to the tribe project for GKE SA.

@askoriy the name you picked `external-project-iam` we will changed to `external-project-iam-roles`, is that okay?

Comment (askoriy) [reply modules/external-project-iam/vars.tf]: yes,. its fine

Comment (anakovt) [reply modules/external-project-iam/vars.tf]: okay, cool
I can change `external-project-iam` -> `external-project-iam-roles` and `gke_tribe_iam_roles` -> `gke_parent_iam_roles`
any other changes you'd like me to do?

Comment (anakovt) [reply modules/external-project-iam/vars.tf]: @askoriy , @sclausson ?

**Review by Santos92 — APPROVED** [review]
LGTM - I dont think we can generify the naming any more than this as there is no custom naming on the variables.

**Pull request merged**